src/Evolution/Systems/Ccz4/Ricci.hpp

```diff
@@ -0,0 +1,71 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace Ccz4 {
+/// @{
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Computes the spatial ricci tensor
+ *
+ * \details Computes the spatial ricci tensor as:
+ * \f{align}
+ *     R_{ij} &=
+ *       \partial_k \Gamma^m{}_{ij} - \partial_j \Gamma^m{}_{ik} +
+ *       \Gamma^l{}_{ij} \Gamma^m{}_{lk} - \Gamma^l{}_{ik} \Gamma^m{}_{lj}
+ * \f}
+ * where
+ * \f{align}
+ *     \partial_k \Gamma^m{}_{ij} &=
+ *       \partial_k \tilde{\Gamma}^m{}_{ij} +
+ *       2 D_k{}^{ml} (\tilde{\gamma}_{jl} P_i + \tilde{\gamma}_{il} P_j
+ *         - \tilde{\gamma}_{ij} P_l) -
+ *       2 \tilde{\gamma}^{ml} (D_{kjl} P_i + D_{kil} P_j - D_{kij} P_l) -
+ *       \tilde{\gamma}^{ml} (
+ *         \tilde{\gamma}_{jl} \partial_{(k} P_{i)} +
+ *         \tilde{\gamma}_{il} \partial_{(k} P_{j)} -
+ *         \tilde{\gamma}_{ij} \partial_{(k} P_{l)})
+ * \f}
+ * \f$\Gamma^k{}_{ij}\f$, \f$\partial_m \tilde{\Gamma}^k{}_{ij}\f$,
```

Comment:
"where \f$\Gamma^...\f$ is the spatial Christoffel symbol, \f$\partial_m...\f$ is the spetial derivative of the ..." would make reading this a lot easier. Right now I have to match terms between two long lists.